tests/e2e/tests/content-manager/history.spec.ts

```diff
@@ -213,13 +284,6 @@ describeOnCondition(edition === 'EE')('History', () => {
   });
 
   test.describe('Single Type', () => {
-    test.beforeEach(async ({ page }) => {
-      await resetDatabaseAndImportDataFromPath('with-admin.tar');
-      await resetFiles();
-      await page.goto('/admin');
-      await login({ page });
-    });
```

Comment:
Why is this removed? Shouldn't we continue to reset the db and git working tree between tests to ensure the tests are isolated?

Reply:
We had the same beforeEach for single and collection types, so I moved it up to the parent describe, now it runs for everything

Reply:
Oh I see what you did, but is that the same thing? Are you sure that runs for each test inside each nested describe block? I thought it would just run before each nested describe but not each test inside those nested describe blocks.

Reply:
Yes I checked, it wouldn't pass otherwise. I see how it's not super intuitive, lmk if you'd prefer to revert it

Reply:
Ok good to know, fine by me if that's how it works :) thanks